Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -69,6 +75,28 @@ extension SPAppDelegate {
 
         return true
     }
+
+    func presentSearch() {
+        dismissAllModals(animated: false, completion: nil)
+        sidebarViewController.hideSidebar(withAnimation: false)
+        if navigationController.viewControllers.contains(noteListViewController) {
+            navigationController.popToViewController(noteListViewController, animated: false)
+        }
+
+        DispatchQueue.main.async {
+            // Switch from trash to all notes as trash doesn't have search
+            if self.selectedTag == kSimplenoteTrashKey {
+                self.selectedTag = nil
+            }
```

Comment:
Since we're coming from the springboard, and the user is triggering Search, how about if we simply always ensure `All Notes` is selected?

Reply:
I checked this with Sly. It is fine to open the app wherever it was before. If tag was selected, we keep the tag selected. Search is global, so it won't be affected by the selected tag.
The only place that doesn't have search is Trash, that's why we switch it to All Notes.